### PR TITLE
I heard you liked running R

### PR DESCRIPTION
**Pull request by hunterowens**
image

RStudio Sessions run perfectly on localhost!

### CODE REVIEW

**Comment by hunterowens** [comment]
fixes #6 for reference.

**Comment by ian-r-rose** [comment]
Nice! On my phone right now, so can't test myself. Does the proxy work, or are you hitting the rstudio port directly?

**Comment by hunterowens** [comment]
proxy works...

flow is new -> rstudio session -> rstudio launches

File: image/Dockerfile [hunk 1]
```diff
@@ -55,7 +55,27 @@ RUN pip install \
...
 
 # AWS CLI is kind of a pain, as it has some old, hard dependencies.
-run pip install awscli --ignore-installed
+RUN pip install awscli --ignore-installed
+
+
+# Rstudio integration 
+
+# install nbrsessionproxy extension
+RUN conda install -yq -c conda-forge && \
```
Comment (ian-r-rose) [inline image/Dockerfile]: This line is currently broken because nothing is installed.

File: image/Makefile [hunk 2]
```diff
@@ -17,3 +17,7 @@ publish: buildlog.txt
 
 clean:
 	rm buildlog.txt
+
+test:
```
Comment (ian-r-rose) [inline image/Makefile]: :+1:

File: image/Dockerfile [hunk 3]
```diff
@@ -55,7 +55,27 @@ RUN pip install \
...
+    conda clean -tipsy
+
+# install rstudio-server
+USER root
+RUN apt-get update && \
```
Comment (ian-r-rose) [inline image/Dockerfile]: I wonder if this should be consolidated with the other `apt` invocation above.

File: image/Dockerfile [hunk 4]
```diff
@@ -55,7 +55,27 @@ RUN pip install \
...
+    curl --silent -L --fail https://download2.rstudio.org/rstudio-server-1.1.419-amd64.deb > /tmp/rstudio.deb && \
+    echo '24cd11f0405d8372b4168fc9956e0386 /tmp/rstudio.deb' | md5sum -c - && \
+    apt-get install -y /tmp/rstudio.deb && \
+    rm /tmp/rstudio.deb && \
+    apt-get clean
+ENV PATH=$PATH:/usr/lib/rstudio-server/bin
+USER $NB_USER
+
+RUN pip install jupyter-rsession-proxy
```
Comment (ian-r-rose) [inline image/Dockerfile]: Likewise this with the above `pip` invocation.

**Comment by ian-r-rose** [comment]
Beautiful:
image

I can also add an item to the JupyterLab launcher to invoke this.

File: image/Dockerfile [hunk 5]
```diff
@@ -75,9 +71,6 @@ RUN apt-get update && \
 ENV PATH=$PATH:/usr/lib/rstudio-server/bin
 USER $NB_USER
 
-RUN pip install jupyter-rsession-proxy
```
Comment (ian-r-rose) [inline image/Dockerfile]: We still need this one.

**Pull request merged**